src/block-components/alignment/edit.js

```diff
@@ -346,6 +348,18 @@ export const Edit = props => {
 	)
 }
 
+// Add additional classes when browser is Firefox to fix alignment
+addFilter( 'stackable.block-components.block-div.classnames', 'alignment-editor-has-polyfill', classes => {
+	const userAgent = navigator.userAgent
+	if ( userAgent.indexOf( 'Firefox' ) !== -1 ) {
```

Comment:
You can move this outside the filter so you can do the `addFilter` only when we're using Firefox. Also make sure that this doesn't error when `navigator` is not present